### PR TITLE
move everything to the CLD2 namespace, update gemspec

### PR DESCRIPTION
**Pull request by craig-day**
Move everything to the CLD2 namespace. Update the gemspec. Remove original forks specific things like build status. 

/cc @jcheatham @grosser 
### Risks
- None.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,7 +18,7 @@ CLD.detect_language("plus ça change, plus c'est la même chose")
 Add this line to your application's Gemfile:
 
 ```ruby
-gem 'cld2', require 'cld'
+gem 'ruby_cld2', require 'cld'
```
Comment (grosser) [inline README.md:21]: why the ruby_ prefix ?

**Comment by grosser** [comment]
why can our stuff not be merged into the original ?
would it make sense to keep the namespace to be able to rebase more easily ?

File: README.md (hunk 1 above)
Comment (craig-day) [inline README.md:21]: Mostly if we release it to release it to rubygems, the name isn't taken yet.

**Comment by craig-day** [comment]
The authors of the original were unresponsive for ~8 months.  We are continually seeing tickets about incorrect language detection, so I was hoping we could get this going so it is easier to maintain/expand. 

As far as the namespacing, we could keep the original.  I changed it because it makes more sense to me and is then not ambiguous about what is coming from where.  That was mostly just my personal preference.

**Comment by grosser** [comment]
Did you try reaching out to the cld2 owners ?

**Comment by grosser** [comment]
sent an email to the cld and cld2 owners ...

**Comment by craig-day** [comment]
I pinged them a few times on the PR I originally made
https://github.com/BanjoInc/cld2/pull/1

I also sent an email to the owners in the gemspec, no response.  Maybe I'll
send another one.

On Wed, Aug 19, 2015 at 2:00 PM, Michael Grosser notifications@github.com
wrote:

> Did you try reaching out to the cld2 owners ?
> 
> On Wed, Aug 19, 2015 at 1:57 PM, Craig Day notifications@github.com
> wrote:
> 
> > The authors of the original were unresponsive for ~8 months. We are
> > continually seeing tickets about incorrect language detection, so I was
> > hoping we could get this going so it is easier to maintain/expand.
> > 
> > As far as the namespacing, we could keep the original. I changed it
> > because it makes more sense to me and is then not ambiguous about what is
> > coming from where. That was mostly just my personal preference.
> > 
> > —
> > Reply to this email directly or view it on GitHub
> > https://github.com/zendesk/cld2/pull/1#issuecomment-132781405.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/zendesk/cld2/pull/1#issuecomment-132782120.

## 

_Craig Day_
Sustaining Engineer
Zendesk http://zendesk.com
Skype: zendesk.cday

**Comment by craig-day** [comment]
Oh... you beat me to it. I guess I won't send them another one.

**Comment by grosser** [comment]
if that does not work out, I'd still prefer to keep the namespace/code as close as possible and go with cld3 or something like that :D

**Pull request closed without merging**